_data/changelogs/select.yml

```diff
@@ -24,3 +24,8 @@ items:
   githubPr: 4656
   githubRepo: uswds
   versionUswds: 3.0.0
+- date: 2022-04-28
+  summary: Renamed component from "dropdown" to "select".
```

Comment:
```suggestion
  summary: Renamed component from `dropdown` to `select`.
```

Reply:
As discussed, going to leave these items with double quotes because it is referencing the name of the component, not just the package.